### ISSUE TITLE
DS-Algo in C++

### ISSUE THREAD

**Issue by Dhruv-VINT** [rishabhgupta03/Hacktoberfest2020#81]
Hey @rishabhgupta03 ! Would like to contribute a few algorithms in C++
**Suggestion:** I think adding folders like C++, Python, Java and in those folders adding respective algos would a nice & clean idea

- ### C++
           Sorting algos
           Searching algos
           Data structures etc

- ### Python
            " "

- ### Java
           " "

and so on....

**Comment by rishabhgupta03** [comment]
done, go ahead and contribute as much as you can !

**Comment by Dhruv-VINT** [comment]
Thanks @rishabhgupta03  I'll get on with it right away !

**Comment by EmmadiDivyaSrujana** [comment]
Hello, @rishabhgupta03 .Can I contribute to DS algo in Cpp?

**Comment by kesari007** [comment]
Hi @rishabhgupta03 can we also add algorithms in cpp?

**Comment by rishabhgupta03** [comment]
@EmmadiDivyaSrujana sure, i am assigning you too this task. just check the questions don't repeat.

**Comment by rishabhgupta03** [comment]
@kesari007 yes you can, all kind of data structure/algorithms are welcomed. please contribute in the specified folder or create one according to need.

**Comment by EmmadiDivyaSrujana** [comment]
Thank you @rishabhgupta03.

**Comment by Anjalikamath** [comment]
Hello @rishabhgupta03 , Can I contribute to DS - Algo in C++?

**Comment by rishabhgupta03** [comment]
sure @Anjalikamath . contribute as much as you can

**Comment by Anjalikamath** [comment]
Thank You @rishabhgupta03
